src/base/ProtocolFees.sol

```diff
@@ -88,10 +81,8 @@ abstract contract ProtocolFees is Owned {
             if (feeOutput.amount > tokenValue.mulDivDown(MAX_FEE_BPS, BPS)) {
                 revert FeeTooLarge(feeOutput.token, feeOutput.amount, feeOutput.recipient);
             }
-            newOutputs[outputsLength + i] = feeOutput;
-
             unchecked {
-                i++;
+                newOutputs[outputsLength + i] = feeOutput;
```

Comment:
curious what the benefit is from moving this assingment into unchecked?

Reply:
the `outputsLength + i`